components/cluster/cluster-glue/cluster-glue.p5m

```diff
@@ -0,0 +1,206 @@
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/include/clplumbing/GSource.h
+file path=usr/include/clplumbing/GSource_internal.h
+file path=usr/include/clplumbing/Gmain_timeout.h
+file path=usr/include/clplumbing/apphb_cs.h
+file path=usr/include/clplumbing/base64.h
+file path=usr/include/clplumbing/cl_log.h
+file path=usr/include/clplumbing/cl_misc.h
+file path=usr/include/clplumbing/cl_pidfile.h
+file path=usr/include/clplumbing/cl_plugin.h
+file path=usr/include/clplumbing/cl_poll.h
+file path=usr/include/clplumbing/cl_quorum.h
+file path=usr/include/clplumbing/cl_quorumd.h
+file path=usr/include/clplumbing/cl_random.h
+file path=usr/include/clplumbing/cl_reboot.h
+file path=usr/include/clplumbing/cl_signal.h
+file path=usr/include/clplumbing/cl_syslog.h
+file path=usr/include/clplumbing/cl_tiebreaker.h
+file path=usr/include/clplumbing/cl_uuid.h
+file path=usr/include/clplumbing/coredumps.h
+file path=usr/include/clplumbing/cpulimits.h
+file path=usr/include/clplumbing/ipc.h
+file path=usr/include/clplumbing/loggingdaemon.h
+file path=usr/include/clplumbing/longclock.h
+file path=usr/include/clplumbing/lsb_exitcodes.h
+file path=usr/include/clplumbing/md5.h
+file path=usr/include/clplumbing/mkstemp_mode.h
+file path=usr/include/clplumbing/netstring.h
+file path=usr/include/clplumbing/proctrack.h
+file path=usr/include/clplumbing/realtime.h
+file path=usr/include/clplumbing/replytrack.h
+file path=usr/include/clplumbing/setproctitle.h
+file path=usr/include/clplumbing/timers.h
+file path=usr/include/clplumbing/uids.h
+file path=usr/include/heartbeat/compress.h
+file path=usr/include/heartbeat/glue_config.h
+file path=usr/include/heartbeat/ha_msg.h
+file path=usr/include/heartbeat/lrm/lrm_api.h
+file path=usr/include/heartbeat/lrm/lrm_msg.h
+file path=usr/include/heartbeat/lrm/racommon.h
+file path=usr/include/heartbeat/lrm/raexec.h
+file path=usr/include/pils/generic.h
+file path=usr/include/pils/interface.h
+file path=usr/include/pils/plugin.h
+file path=usr/include/stonith/expect.h
+file path=usr/include/stonith/st_ttylock.h
+file path=usr/include/stonith/stonith.h
+file path=usr/include/stonith/stonith_config_xml.h
+file path=usr/include/stonith/stonith_expect_helpers.h
+file path=usr/include/stonith/stonith_plugin.h
+file path=usr/include/stonith/stonith_plugin_common.h
+file path=usr/include/stonith/stonith_signal.h
+
+file path=usr/lib/heartbeat/base64_md5_test mode=0555
+file path=usr/lib/heartbeat/ha_logd mode=0555
+file path=usr/lib/heartbeat/ipctest mode=0555
+file path=usr/lib/heartbeat/ipctransientclient mode=0555
+file path=usr/lib/heartbeat/ipctransientserver mode=0555
+file path=usr/lib/heartbeat/logtest mode=0555
+file path=usr/lib/heartbeat/lrmd mode=0555
+file path=usr/lib/heartbeat/plugins/InterfaceMgr/generic.la mode=0555
```

Comment:
Transform will drop *\.la$ files anyway, but better avoid listing them in manifest